kernel/base/src/main/java/com/twosigma/beakerx/kernel/commands/MagicCommand.java

```diff
@@ -190,6 +204,36 @@ private MagicCommandFunctionality classpathAddJar() {
     };
   }
 
+  private void validateWildcardPath(String path) {
+    if (!path.matches("^([^*]*\\*){1}[^*]*$") || !path.endsWith("*")) {
```

Comment:
@scottdraves regex check if theres only one * symbol in path, and after OR statement i'm checking if statement is ended with *